dev-docs/modules/userId.md

```diff
@@ -1156,9 +1157,6 @@ pbjs.setConfig({
             }
         },{
             name: "pubCommonId",
```

Comment:
Just verifying - the name isn't changing here?

Reply:
this is where we will use alias, the name will change to "sharedid" but if "pubCommonId" is referenced it will be aliased to "sharedid"